portal-kernel/src/com/liferay/portal/kernel/portlet/BasePortletLayoutFinder.java

```diff
@@ -120,12 +116,14 @@ public Result find(ThemeDisplay themeDisplay, long groupId)
 	protected String getPortletId(
 		LayoutTypePortlet layoutTypePortlet, String portletId) {
 
-		for (String curPortletId : layoutTypePortlet.getPortletIds()) {
+		for (Portlet curPortletId :
```

Comment:
`curPortletId` is no longer an ID, it is a `Portlet`. Can you rename the variable to match the type?

Reply:
Sure, I fixed the commit and resent in #4341